app/views/pages/home.html.haml

```diff
@@ -16,7 +16,7 @@
           .hero__inline-image
             = image_tag "registers-top-image.png", role: "presentation"
           %p.hero__description
-            GOV.UK Registers helps government design and build services on a consistent and high quality data infrastructure via our secure API.
+            GOV.UK Data Registers helps government design and build services on a consistent and high quality data infrastructure via our secure API.
```

Comment:
Since these changes came directly from the project manager, I don't know what process this went through before being raised with me. Happy to park this and other similar changes until that discussion has taken place.

Reply:
I've updated this pull request with the updated set of changes in the card.